workflow/commit-messages.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Commit guidelines
+description: Some general guidelines to help you in writing good commits for collaboration and clarity.
+---
+
+This document was written as an overview of how an optimal commit should be written. But in the end it is still your decision , no Pull Request should be stuck in a review phase because its commits differ from the practices described below.
+But if you are unsure of how to write them, thos should give you a good starting point for great collaboration.
+
+As a general guideline you can follow the idea that _a single commit should enclose a single problem/feature that has been fixed/added_.
+
+This way of organizing your changes to the codebase makes is easy for reviewers and people who try to understand the history of a project to determine what has been happening in the project.
+
+Here a both a good and a bad example of of commit message.
+
+**Bad:**
+
+```
+fixes bug
+```
+
+**Good:**
+
+```
+Fix bug where database gets deleted on POST request
+```
+
+## Breaking up large tickets
+
+If your feature/ticket includes mutliple things that you have to complete in order to submit a pull request, these should all be seperate commit messages, according to the general guideline above. This will make it easy for the reviewer to understand what exactly has happened.
+But make sure that this is a something that is done throughout your project. If another workflow is used, such as squashing all your commits together before you merge your branch into master this is not necessary. Just make sure that your commit message for the squash commit includes a detailed message of the changes.
+
```

Comment:
Might be good to mention here that it is fine to have multiline commits (in the sqashed commits workflow) in the form, ensuring all the changes are enumerated:

  _Overall commit message._

  * _Change 1_
  * _Change 2_

Reply:
What do you think of the content I have added @squidsoup @nottrobin . Should it be be more explicit to use multiline commit messages?